cypress/e2e/immutableDatabaseTests/linkCheck.ts

```diff
@@ -0,0 +1,116 @@
+import { setTokenUserViewPort } from '../../support/commands';
+
+// Formats paths to be displayed in an error log
+function formatPaths(paths: string[]): string {
+  let formattedPaths = '';
+  paths.forEach((path) => {
+    formattedPaths += '\n- ' + path;
+  });
+  return formattedPaths;
+}
+
+describe('Find broken anchor links', () => {
+  setTokenUserViewPort();
+
+  let visitedUrls: string[] = [
+    'https://twitter.com/DockstoreOrg', // Skip, test only fails because it redirects to login page if not logged in
+  ];
+
+  function isDynamicUrl(url: string): boolean {
+    return url.includes('/my-workflows/') || url.includes('/my-tools/');
+  }
+
+  function checkUrls(path: string) {
+    let brokenUrls = [];
+    cy.visit(path).wait(500); // Temporary solution to ensure the page loads entirely
+    cy.get('a')
+      .each((anchor) => {
+        cy.get(anchor).then((anchor) => {
+          const href = anchor.prop('href');
+          // Send requests to non-dynamic URLs that haven't yet been visited
+          if (href && !visitedUrls.includes(href) && !isDynamicUrl(href)) {
+            cy.request({
+              url: href,
+              failOnStatusCode: false,
+            }).then((result) => {
+              if (result.status != 200) {
+                brokenUrls.push(href);
+                cy.log(`${result.status}: ${href}`);
+              } else {
+                visitedUrls.push(href); // Add successful links to visitedUrls so that they won't be visited again
+              }
```

Comment:
This code will only add the checked link to `visitedUrls` after a successful visit, so broken links might be checked repeatedly.  That behavior is fine, given that most links aren't broken, but curious if there was a reason for it.

Reply:
I think it would help us report every instance of a broken link, so that we know all the places to fix it. Like if https://docs.dockstore.org/en/stable/advanced-topics/advanced-features.html were coded manually onto multiple pages, it'll remind us to change the link on every page because tests for each page would fail separately.

Reply:
yeah, that makes sense, thanks!